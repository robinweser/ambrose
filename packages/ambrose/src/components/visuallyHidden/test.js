import React from 'react'

import VisuallyHidden from '.'

import createSnapshot from '../../testing/createSnapshot'

describe('<VisuallyHidden />', () => {
  it('renders correctly with default props', () => {
    const snapshot = createSnapshot(<VisuallyHidden>Children</VisuallyHidden>)
    expect(snapshot).toMatchSnapshot()
  })
})
