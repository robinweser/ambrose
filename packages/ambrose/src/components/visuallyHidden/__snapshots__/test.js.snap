// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<VisuallyHidden /> renders correctly with default props 1`] = `
".cls_0 {
  display: contents;
}
.cls_1 {
  clip: rect(1px, 1px, 1px, 1px);
  -webkit-clip-path: inset(50%);
  clip-path: inset(50%);
  position: absolute;
  overflow: hidden;
  white-space: nowrap;
  border: 0;
  padding: 0;
  height: 1px;
  width: 1px;
}


<div className=test-theme cls_0>
  <div className=cls_1>Children</div>
</div>;
"
`;
